tyr/security_groups/management.py

```diff
@@ -0,0 +1,35 @@
+rule = 'management'
+
+definition = {
+    'rules': [
+        {
+            'port': 22,
+            'source': [
+                'p-ops-vpn',
+                '@hudl-lincoln-east',
+                '@hudl-lincoln-west',
+                '@hudl-boston'
+                '@hudl-omaha'
+            ]
+        },
+        {
+            'port': 3389,
```

Comment:
So, technically comments are supported. You can just put Python comments in the middle of dictionaries

```
>>> d = {
...     'hello': 'world',
...     # testing
...     'foo': 'bar'
... }
>>> 
>>> d['hello']
'world'
>>> d['foo']
'bar'
```

but they're not in place. But more than adding comments, I definitely like the idea of constants for ports.